_includes/accessibility-tests/test-results-summary.html

```diff
@@ -0,0 +1,142 @@
+            {% assign status_icon = pass_icon %}
+            {% assign icon_color = "pass" %}
+          {% elsif status == "Passed with exceptions" %}
+            {% assign status_icon = exceptions_icon %}
+            {% assign icon_color = "exception" %}
+          {% elsif status == "Conditional" %}
+            {% assign status_icon = conditional_icon %}
+            {% assign icon_color = "conditional" %}
+          {% elsif status == "Failed" %}
+            {% assign status_icon = fail_icon %}
+            {% assign icon_color = "fail" %}
+          {% endif %}
+          <th>
+            <div class="display-flex flex-align-center">
+                <svg class="usa-icon site-checklist-icon--{{ icon_color | downcase }}" aria-hidden="true" role="img">
+                  <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#{{ status_icon }}"></use>
+                </svg>
+              {{ status }}
+            </div>
+          </th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>{{ items_total }}</td>
+        {% for status in status_types %}
+          {% if status == "Passed" %}
+            {% assign item_count = items_pass %}
+          {% elsif status == "Passed with exceptions" %}
+            {% assign item_count = items_exception %}
+          {% elsif status == "Failed" %}
+            {% assign item_count = items_fail %}
+          {% elsif status == "Conditional" %}
+            {% assign item_count = items_conditional %}
+          {% endif %}
+          <td class="border-{{ borderColor }}">{{ item_count | default:0 }}</td>
+        {% endfor %}
+      </tr>
+    </tbody>
+  </table>
+
+  <!-- Add test summary for mobile widths -->
+  <div class="site-checklist-status-list">
+    <p class="site-checklist-status-list__summary">Overview of recent accessibility test results:</p>
+    <ul class="usa-icon-list margin-top-0">
+      {% for status in status_types %}
+        {% if status == "Passed" %}
+          {% assign status_icon = pass_icon %}
+          {% assign icon_color = "pass" %}
+          {% assign item_count = items_pass %}
+          {% assign status_text = "Passed" %}
+        {% elsif status == "Passed with exceptions" %}
+          {% assign status_icon = exceptions_icon %}
+          {% assign icon_color = "exception" %}
+          {% assign item_count = items_exception %}
+          {% assign status_text = "Passed with exceptions" %}
+        {% elsif status == "Failed" %}
+          {% assign status_icon = fail_icon %}
+          {% assign icon_color = "fail" %}
+          {% assign item_count = items_fail %}
+          {% assign status_text = "Failed" %}
+        {% elsif status == "Conditional" %}
+          {% assign status_icon = conditional_icon %}
+          {% assign icon_color = "conditional" %}
+          {% assign item_count = items_conditional %}
+          {% assign status_text = "Conditional" %}
+        {% endif %}
```

Comment:
Possible duplicate of line 50-63?

Reply:
I couldn't figure out how to make this less duplicative. I'd like to add it as a possible follow-on task.

Reply:
Addressed in #2438.